frontend/agave.jsx

```diff
@@ -5,7 +5,24 @@ import configureStore from './store/store';
 
 
 document.addEventListener('DOMContentLoaded', () => {
-  const store = configureStore();
+  let preloadedState;
+
+  if (localStorage['currentUser']) {
+    const data = JSON.parse(localStorage['currentUser']);
+
+    preloadedState = {
```

Comment:
can this be factored out into a separate function that returns `preloadedState` or `null`?

Reply:
or `{}` instead of `null`